src/pages/lib/storage/transfer-acceleration/q/platform/[platform].mdx

```diff
@@ -0,0 +1,8 @@
+export const meta = {
+  title: `Use Transfer Acceleration`,
+  description: `How to enable and use S3's Transfer Acceleration on your S3 bucket`,
+};
+
+import js from "/src/fragments/lib/storage/js/transfer-acceleration.mdx";
```

Comment:
can you also add the fragment for React Native here? I can only see the JS one.

Reply:
Since the underlying code snippets are the same across JS n RN, I have imported the same fragment and conditionally render it for both JS n RN. An example of a similar conditional import can also be found [here](https://github.com/aws-amplify/docs/blob/3cb94fb9b33e2ff5eba509dfa561057b92ee8324/src/pages/lib/datastore/advanced-workflows/q/platform/%5Bplatform%5D.mdx?plain=1#L20)